CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Bugfix: Fixed being unable to see the usercard of VIPs who have Asian language display names. (#4174)
```

Comment:
This is the less-correct but more friendly to the average user, changelog.

I'd be willing to take the more-correct but less friendly approach as well:
```suggestion
- Bugfix: Prevented using display names for the `/vips` command, as Twitch does not allow looking up a user by their Asian language name. (#4174)
```